src/ringmesh/basic/command_line.cpp

```diff
@@ -55,6 +55,8 @@ namespace RINGMesh
                 GEO::CmdLine::ARG_ADVANCED );
             GEO::CmdLine::declare_arg( "algo:tet", "TetGen",
                 "Toggles the tetrahedral mesher (TetGen, MG_Tetra)" );
+            GEO::CmdLine::declare_arg( "sys:plugins", "",
```

Comment:
No need to give paths just names?

Reply:
Yes. Paths will be found using OS library detection (RPATH, PATH, LD_LIBRARY_PATH ...)